tests/SentryLink.test.ts

```diff
@@ -29,11 +29,11 @@ describe('SentryLink', () => {
   beforeEach(() => {
     testkit.reset();
 
-    Sentry.configureScope((scope) => {
-      scope.clearBreadcrumbs();
-      scope.setTransactionName();
-      scope.setFingerprint([]);
-    });
+    const scope = Sentry.getCurrentScope();
+
+    scope.clearBreadcrumbs();
```

Comment:
The [`addBreadcrumb` function adds breadcrumbs to the isolation scope](https://github.com/getsentry/sentry-javascript/blob/8.2.1/packages/core/src/breadcrumbs.ts#L17-L40) so you'll need clear them from there instead

```
Sentry.getIsolationScope().clearBreadcrumbs();
```

Reply:
Well that was non-obvious, but seems to fix the tests, nice